GitOut/Features/Wpf/AttachedProperties/TextZoom.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace GitOut.Features.Wpf.AttachedProperties
+{
+    public class TextZoom
+    {
+        private static readonly double Scale = .2;
+
+        public static KeyGesture? GetZoomInKeyGesture(DependencyObject obj) => (KeyGesture?)obj.GetValue(ZoomInKeyGestureProperty);
```

Comment:
Not sure but these should be below properties I guess (properties over methods)?